src/validation/obfl.rng

```diff
@@ -549,6 +558,15 @@
         </choice>
       </attribute>
     </optional>
+    <optional>
+      <attribute name="mark-capital-letters">
+        <choice>
```

Comment:
Maybe the time has come to extract the attributes `translate`, `hyphenate` and `mark-capital-letters` (but not `xml:lang`) into a separate group, to get rid of this duplication.